lib/wasix/src/bin_factory/binary_package.rs

```diff
@@ -224,6 +224,25 @@ impl BinaryPackage {
             }
         })
     }
+
+    pub fn infer_entrypoint(&self) -> Result<&str, anyhow::Error> {
+        if let Some(entrypoint) = self.entrypoint_cmd.as_deref() {
+            return Ok(entrypoint);
+        }
+
+        match self.commands.as_slice() {
+            [] => anyhow::bail!("The WEBC file doesn't contain any executable commands"),
+            [one] => Ok(one.name()),
+            [..] => {
+                let mut commands: Vec<_> = self.commands.iter().map(|cmd| cmd.name()).collect();
+                commands.sort();
+                anyhow::bail!(
+                    "Unable to determine the WEBC file's entrypoint. Please choose one of {:?}",
+                    commands,
```

Comment:
This is not a good message. Users should not care a bit about webc.

```suggestion
                anyhow::bail!(
                    "Unable to determine the package's entrypoint. Please choose one of {:?}",
                    commands,
```